### DISCUSSION TITLE
Is it possible to perform the Quarto compilation in two steps on Github?

### DISCUSSION THREAD

**Discussion by shravanngoswamii** [quarto-dev/quarto-cli#9730] [Q&A] [answered]
step 1: on the master branch, compile all qmd notebooks to markdown files in parallel, then push them to a markdown branch
step 2: on the markdown branch, compile a Quarto project referencing all these markdown files, instead of original qmd files
step 3: push the output of step 2 to branch gh-pages

Do we lose any formatting information when adopting this qmd --> markdown --> HTML workflow instead of qmd --> HTML?

**Comment by cscheid** [comment, marked as answer]
It's not currently possible.

**Reply by shravanngoswamii** [reply]
ok
